src/css/base/_variables.scss

```diff
@@ -1,5 +1,5 @@
 //typography
-@import url('https://fonts.googleapis.com/css?family=Lora|Open+Sans:400,600,700,800');
+@import url('https://fonts.googleapis.com/css?family=Lora:400,400i|Open+Sans:400,600,700,800&text=Welcome%20to');
```

Comment:
what does this `text=` param do? I've never seen that.

Reply:
https://developers.google.com/fonts/docs/getting_started#optimizing_your_font_requests
"Oftentimes, when you want to use a web font on your website or application, you know in advance which letters you'll need. This often occurs when you're using a web font in a logo or heading.

In these cases, you should consider specifying a text= value in your font request URL. This allows Google to return a font file that's optimized for your request. In some cases, this can reduce the size of the font file by up to 90%."